src/www/ui/api/documentation/openapi.yaml

```diff
@@ -7,7 +7,7 @@ openapi: 3.0.2
 info:
   title: FOSSology API
   description: Automate your fossology instance using REST API
-  version: 1.5.1
+  version: 1.5.2
```

Comment:
As discussed, please avoid updating version in PRs.

Reply:
Done @GMishx